python/ray/tune/tests/tutorial.py

```diff
@@ -7,25 +7,102 @@
 import torch
 import torch.optim as optim
 from torchvision import datasets
+from torch.utils.data import DataLoader
 
 from ray import tune
 from ray.tune.schedulers import ASHAScheduler
-from ray.tune.examples.mnist_pytorch import get_data_loaders, ConvNet, train, test
 # __tutorial_imports_end__
 # yapf: enable
 
 
 # yapf: disable
+# __model_def_begin__
+class ConvNet(nn.Module):
+    def __init__(self):
+        super(ConvNet, self).__init__()
+        self.conv1 = nn.Conv2d(1, 3, kernel_size=3)
+        self.fc = nn.Linear(192, 10)
+
+    def forward(self, x):
+        x = F.relu(F.max_pool2d(self.conv1(x), 3))
+        x = x.view(-1, 192)
+        x = self.fc(x)
+        return F.log_softmax(x, dim=1)
+# __model_def_end__
```

Comment:
This block (`__model_def_*`) is currently not referenced in the `tune-tutorial.rst` (or anywhere else). Is this on purpose?

Reply:
fixed!